source/includes/_tipser-elements.md

```diff
@@ -132,16 +133,29 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
 
 ## `Collection` Element
 
-You can your store in collections. Each collection however can be rendered separately as a get the look widget or a simple product group, depending on your needs. It is all possible through `Collection` Element.
+You can render your collections as `Store`. Each collection, however, can be rendered separately as a simple product group, depending on your needs. It is all possible through the `Collection` Element.
```

Comment:
Better but still a bit confusing. Do we really need to refer to store component in a section related to collection? Can't we just focus on what a collection is?